Assets/Plugins/Source/Editor/Utility/GUIEditorUtility.cs

```diff
@@ -696,7 +701,7 @@ private static void RenderSetOfNamed<T>(
             Action<Rect, Named<T>> renderItemFn,
             Action addNewItemFn,
             Action<Named<T>> removeItemFn
-        ) where T : IEquatable<T>
+        ) where T : IEquatable<T>, new()
```

Comment:
Yep! requiring `new()` means that it must have a parameter-less constructor. Because by definition `struct` objects satisfy that requirement, it works for them as well.